index.js

```diff
@@ -259,12 +260,32 @@ Converter.prototype.buildTypeProperties = function(oldType) {
  * @returns {object} - Swagger 2.0 equivalent
  */
 Converter.prototype.buildDataType = function(oldDataType) {
-  if (!oldDataType) { return; }
+  if (!oldDataType) { return {}; }
   assert(typeof oldDataType === 'object');
 
-  var result = this.buildTypeProperties(
-    oldDataType.type || oldDataType.dataType || oldDataType.responseClass
-  );
+  var oldType =
+    oldDataType.type || oldDataType.dataType || oldDataType.responseClass;
+  var oldItems = oldDataType.items;
+
+  if (isValue(oldType)) {
+    //handle List[<TYPE>] types from 1.1 spec
+    var match = oldType.match(/^(.*)\[(.*)\]$/);
```

Comment:
I update comment message.
It pretty simple RegEx, see:
http://regexper.com/#%5E(.*)%5C%5B(.*)%5C%5D%24